rpcs3/Emu/RSX/Common/FragmentProgramDecompiler.cpp

```diff
@@ -255,6 +260,24 @@ std::string FragmentProgramDecompiler::Format(const std::string& code)
 		{ "$_c", std::bind(std::mem_fn(&FragmentProgramDecompiler::AddConst), this) }
 	};
 
+	if (!ignore_redirects)
+	{
+		//Special processing redirects
+		switch (dst.opcode)
+		{
+		case RSX_FP_OPCODE_TEXBEM:
+		case RSX_FP_OPCODE_TXPBEM:
+		{
+			//Redirect parameter 0 to the x2d temp register for TEXBEM
+			//TODO: Organize this a little better
+			std::pair<std::string, std::string> repl[] = { { "$0", "x2d" } };
+			std::string result = fmt::replace_all(code, repl);
```

Comment:
is this used anywhere?

Reply:
copypasta, should have been used in the next line instead of code var